frontend/ui/widget/doublespinwidget.lua

```diff
@@ -77,14 +77,15 @@ function DoubleSpinWidget:init()
             },
         }
     end
+
+    -- Actually the widget layout
     self:update()
 end
 
 function DoubleSpinWidget:update()
-    -- This picker_update_callback will be redefined later. It is needed
-    -- so we can have our MovableContainer repainted on NumberPickerWidgets
-    -- update. It is needed if we have enabled transparency on MovableContainer,
-    -- otherwise the NumberPicker area gets opaque on update.
+    -- This picker_update_callback will be redefined later.
+    -- It's a hack to restore transparency after a Button unhighlight in NumberPicker,
+    -- in case the MovableContainer was actually made transparent, and Button highlights are enabled.
```

Comment:
Yeah, the fact that NumberPicker isn't re-initialized every update means the only thing that get redrawn are the button w/ flash_ui.

.... and possibly the center value button, now that I think about it, because I fixed it w/ flash_ui, but I may have forgotten to check without.

Reply:
Hmm, actually is the reverse, it works for the center button because it just pops the VK, but the buttons do refresh the center button, and that resets the alpha ^^.

Reply:
(And now I understand why the hack was disabled in the center button before \o/).